src/main/java/com/conveyal/gtfs/loader/Table.java

```diff
@@ -155,9 +156,9 @@ public Table (String name, Class<? extends Entity> entityClass, Requirement requ
     public static final Table SHAPES = new Table("shapes", ShapePoint.class, OPTIONAL,
             new StringField("shape_id", REQUIRED),
             new IntegerField("shape_pt_sequence", REQUIRED),
-            new DoubleField("shape_pt_lat", REQUIRED, -80, 80),
-            new DoubleField("shape_pt_lon", REQUIRED, -180, 180),
-            new DoubleField("shape_dist_traveled", OPTIONAL, 0, Double.POSITIVE_INFINITY),
+            new DoubleField("shape_pt_lat", REQUIRED, -80, 80, 6),
+            new DoubleField("shape_pt_lon", REQUIRED, -180, 180, 6),
+            new DoubleField("shape_dist_traveled", OPTIONAL, 0, Double.POSITIVE_INFINITY, 2),
```

Comment:
Oh interesting, I just assumed the units were meters, but I didn't realize they could be literally any kind of distance measure.  In this case, we shouldn't have any limitation on precision for this field because it is theoretically possible to use Astronomical Units.